data/enterprise-nav-data.json

```diff
@@ -288,25 +288,45 @@
-      { "title": "v202201-1", "path": "release/v202201-1" },
-      { "title": "v202112-2", "path": "release/v202112-2" },
-      { "title": "v202112-1", "path": "release/v202112-1" },
-      { "title": "v202111-1", "path": "release/v202111-1" },
-      { "title": "v202110-1", "path": "release/v202110-1" },
-      { "title": "v202109-2", "path": "release/v202109-2" },
-      { "title": "v202109-1", "path": "release/v202109-1" },
-      { "title": "v202108-1", "path": "release/v202108-1" },
-      { "title": "v202107-1", "path": "release/v202107-1" },
-      { "title": "v202106-1", "path": "release/v202106-1" },
-      { "title": "v202105-1", "path": "release/v202105-1" },
-      { "title": "v202104-1", "path": "release/v202104-1" },
-      { "title": "v202103-3", "path": "release/v202103-3" },
-      { "title": "v202103-2", "path": "release/v202103-2" },
-      { "title": "v202103-1", "path": "release/v202103-1" },
-      { "title": "v202102-2", "path": "release/v202102-2" },
-      { "title": "v202102-1", "path": "release/v202102-1" },
-      { "title": "v202101-1", "path": "release/v202101-1" }
+      { "title": "2022", "routes": [
+          { "title": "Overview", "path": "release/2022" },
+          { "title": "v202201-2", "path": "release/2022/v202201-2" },
+          { "title": "v202201-1", "path": "release/2022/v202201-1" }
+        ]
+      },
+      { "title": "2021", "routes": [
+          { "title": "Overview", "path": "release/2021" },
+          { "title": "v202112-2", "path": "release/2021/v202112-2" },
+          { "title": "v202112-1", "path": "release/2021/v202112-1" },
+          { "title": "v202111-1", "path": "release/2021/v202111-1" },
+          { "title": "v202110-1", "path": "release/2021/v202110-1" },
+          { "title": "v202109-2", "path": "release/2021/v202109-2" },
+          { "title": "v202109-1", "path": "release/2021/v202109-1" },
+          { "title": "v202108-1", "path": "release/2021/v202108-1" },
+          { "title": "v202107-1", "path": "release/2021/v202107-1" },
+          { "title": "v202106-1", "path": "release/2021/v202106-1" },
+          { "title": "v202105-1", "path": "release/2021/v202105-1" },
+          { "title": "v202104-1", "path": "release/2021/v202104-1" },
+          { "title": "v202103-3", "path": "release/2021/v202103-3" },
+          { "title": "v202103-2", "path": "release/2021/v202103-2" },
+          { "title": "v202103-1", "path": "release/2021/v202103-1" },
+          { "title": "v202102-2", "path": "release/2021/v202102-2" },
+          { "title": "v202102-1", "path": "release/2021/v202102-1" },
+          { "title": "v202101-1", "path": "release/2021/v202101-1" }
+        ]
+      },
+      { "title": "2020", "routes": [
+          { "title": "Overview", "path": "release/2020" }
+        ]
+      },
+      { "title": "2019", "routes": [
+          { "title": "Overview", "path": "release/2019" }
+        ]
+      },
+      { "title": "2018", "routes": [
```

Comment:
I keep going back and forth when I see the sidebar. Part of me wants 2018 and older to just be "Older" since the product predates the current Replicated based setup of these releases and wasn't created in 2018 (though I'm not sure we have the release notes for the discontinued AMI based product). Also because over time this sidebar will continue to grow in the coming years and it'd be nice to keep it from being just a big ole list. 

Then again if we have a plan for rolling off older releases I guess this no longer matters. Also we have plenty of time before this becomes a problem. 😁

Reply:
I'm with you on this. Part of me wants to see "current" and "archived" sections in the sidebar and part of me wants to only show the currently supported releases with some text telling readers to find old release notes in the older versioned docs.

For now I'll leave it as is and we can revisit it once we establish more formalized processes around release notes, versioned documentation, and supported releases.